doc/man/8/ceph.rst

```diff
@@ -369,11 +400,25 @@ Usage::
 
 Subcommand ``new`` to make a new file system using named pools <metadata> and <data>
 
+Subcommand ``lsflags`` displays all the flags set on the given FS.
```

Comment:
This section should go below `ceph fs new` usage ?

Reply:
@kotreshhr I think he has placed everything according to lexicographical order. That's why `lsflags` is above `new`